intent-classifier/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "intent-classifier",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "raw-to-csv": "node training/raw_to_csv.js",
+    "csv-to-tensors": "node training/csv_to_tensors.js",
+    "raw-to-tagged-tokens": "node training/raw_to_tagged_tokens.js",
+    "tokens-to-embeddings": "node training/tokens_to_embeddings.js",
+    "prep-data": "npm-run-all raw-to-csv csv-to-tensors raw-to-tagged-tokens tokens-to-embeddings",
+    "train-intent": "node training/train_intent.js",
+    "train-tagger": "node training/train_tagger.js",
+    "prep-client": "node copy_models.js",
+    "intent-app": "yarn prep-client && parcel app/index.html --open",
+    "tagger-app": "yarn prep-client && parcel app/tagger.html --open"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@tensorflow-models/universal-sentence-encoder": "1.0.1",
+    "@tensorflow/tfjs": "1.0.2",
+    "@tensorflow/tfjs-node": "1.0.2",
+    "@tensorflow/tfjs-node-gpu": "1.0.2",
+    "argparse": "^1.0.10",
```

Comment:
Since this example is on the larger side, let's enable eslint here (reuse as much stuff from tslint config as possible). Then we can use this example as a template on how to add eslint to all other examples, and then we can enable CI